docs/resequencing.md

```diff
@@ -1,122 +1,380 @@
+  variants.
+* `--output_gvcf=/workdir/output/vcf_files/LineA_B_composite_reseq.g.vcf`:
+  The output gVCF file path.
+* `--intermediate_results_dir=/workdir/intermediate_results`: The
+  directory where DeepVariant will write intermediate files generated
+  during the analysis. These files are useful for debugging or
+  optimizing performance, but can be deleted after run if not needed.
+* `--num_shards=4`: This specifies the number of CPU cores or threads
+  to parallelize the job across. In this case, we are using `4` CPU
+  threads. The higher number of shards, the faster the job, but will
+  require more computational resources.
+
+
+
+### Filter variants
+
+To identify rare alleles, the VCF file output by the variant caller 
+should be filtered to select variants based on quality scores, depth, 
+or other measures. The following is an example of how to use 
+[bcftools](https://github.com/samtools/bcftools) to filter the VCF 
+file created above:
+
+```shell
+bcftools view -m2 -M2 -v snps \
+    -i 'QUAL>20 && GT="1/1" && DP>4' \
+    output/vcf_files/LineA_B_composite_reseq.vcf \
+    -o output/vcf_files/LineA_B_composite_reseq_filt.vcf
 ```
-* Align the reads you used in the imputation pipeline against the composite fasta using minimap2 or other aligner
+In summary, this command is using the following bcftools parameters:
+
+* `-m2`: Specifies that only **diploid variants** should be included.
+  The value `2` represents the ploidy level (diploid).
+* `-M2`: Ensures that variants with more than two alleles are
+  excluded. Using this in conjunction with `-m2` ensures that only
+  **biallelic diploid variants** are retained.
+* `-v snps`: This option restricts the output to only **SNPs**.
+* `-i 'QUAL>20 && GT="1/1" && DP>4'`: This specifies the filtering
+  conditions in which variants have the qualities:
+    + `QUAL>20`: only include variants with a **quality score greater
+      than 20**. The quality score indicates the confidence of the
+      variant call, and this filter ensures that only high-confidence
+      variants are retained.
+    + `GT="1/1"`: this ensures that only **homozygous alternate
+      variants** are included (where the genotype is `1/1`, 
+      indicating) both alleles are the alternate allele).
+    + `DP>4`: retain only variants with a **depth of coverage
+      greater than 4**. The depth of coverage (`DP`) indicates the
+      number of reads covering the variant position.
+* `output/vcf_files/LineA_B_composite_reseq.vcf`: the
+  input VCF file that contains the variants to be filtered.
+* `-o output/vcf_files/LineA_B_composite_reseq_filt.vcf`: 
+  the output file where the filtered variants will be saved.
+
+
+
+### Create final VCF in haplotype coordinates
+Now that we have retained "high quality" variants, we can finally
+convert the coordinates from the composite FASTA file to coordinates
+found in the haplotypes within PHG population. To perform this, we
```

Comment:
Do we need to be more explicit that these are not reference coordinates (or coordinates relative to any single assembly)? I could see some confusion occurring here.

Reply:
Updated for clarity